package.json

```diff
@@ -29,7 +29,9 @@
     "ea-start": "./build.js -run -watch --settings ../ForumCredentials/settings-dev.json --mongoUrlFile ../ForumCredentials/dev-db-conn.txt",
     "ea-start-staging-db": "./build.js -run --watch --mongoUrlFile ../ForumCredentials/staging-db-conn.txt --settings ../ForumCredentials/settings-local-dev-staging-db.json",
     "ea-start-prod-db": "./build.js -run --watch --mongoUrlFile ../ForumCredentials/prod-db-conn.txt --settings ../ForumCredentials/settings-local-dev-prod-db.json",
-    "ea-start-memory-db": "./scripts/mongoMemoryServer.js & ./build.js -run --watch --mongoUrl mongodb://localhost:27017 --settings ./cypress/instance-settings-test.json",
+    "ea-start-testing-db": "./build.js -run --mongoUrl ${TESTING_DB_URL:=$(cat ../ForumCredentials/testing-db-conn.txt)} --settings ./settings-test.json",
```

Comment:
Now that you need a foreign db anyway, why not move the settings-test.json file to ForumCredentials?